modules/markup/html.go

```diff
@@ -793,25 +809,36 @@ func fullIssuePatternProcessor(ctx *RenderContext, node *html.Node) {
 	if ctx.Metas == nil {
 		return
 	}
-
 	next := node.NextSibling
 	for node != nil && node != next {
 		m := getIssueFullPattern().FindStringSubmatchIndex(node.Data)
 		if m == nil {
 			return
 		}
+
+		Changedm := getFilesChangedFullPattern().FindStringSubmatchIndex(node.Data)
+		// if the link is from files changed tab in pull requests, leave it as it is
+		if Changedm != nil {
```

Comment:
Hmm, the variable name really looks strange.

I think it could be something like `mChangedTab` at least (ps: I still don't understand what is "changed tab", do other reviewers know?)

And I guess it's better to put a sample URL beside `getFilesChangedFullPattern` to help reviewers to  understand what kind of URLs it matches.

Reply:
It is link copied from the "files changed" tab from pull request page, or the files changed url itself.

<img width="706" alt="截屏2023-03-28 11 00 19" src="https://user-images.githubusercontent.com/17645053/228116627-410c9a29-d49b-4907-8833-391b94a3cd53.png">

I will add the sample to the comment later